bool down
# Pointed down if True, centered if False
---
bool success